csrankings-r.csv

```diff
@@ -8,6 +8,7 @@ R. Clinton Whaley,Indiana University,http://homes.sice.indiana.edu/rcwhaley,NOSC
 R. E. Ellis,Queen’s University,http://www.cs.queensu.ca/~ellis,u97Fg9IAAAAJ
 R. Govindarajan,IISc Bangalore,http://www.csa.iisc.ac.in/~govind,wWrR-qgAAAAJ
 R. Inkulu,IIT Guwahati,http://www.iitg.ernet.in/rinkulu,NOSCHOLARPAGE
+R. Iris Bahar,Brown University,http://cs.brown.edu/people/irisbahar,kfIj9oYAAAAJ

```

Comment:
Please add entries for all aliases as well.

Reply:
I did not mean to `dblp-aliases.csv`, which you cannot alter. I meant add entries in `csrankings-*.csv`.

Reply:
Fixed. Thanks for pointing it out!